trace_decoder/src/lib.rs

```diff
@@ -109,30 +63,395 @@
 //! - if there are withdrawals, one dummy transaction is added at the end, with
 //!   all the withdrawals in it.
 
-#![feature(linked_list_cursors)]
-#![feature(trait_alias)]
-#![feature(iter_array_chunks)]
 #![deny(rustdoc::broken_intra_doc_links)]
-#![deny(missing_debug_implementations)]
-#![deny(missing_docs)]
```

Comment:
These are good to have IMHO, would deserve a TODO to not get forgotten and possibly addressed later.

Reply:
restored to `warn`, they'll fail in CI because we `-D warnings`, but shouldn't hamper local development